lumberjack.go

```diff
@@ -34,10 +34,12 @@ import (
 )
 
 const (
-	backupTimeFormat = "2006-01-02T15-04-05.000"
-	defaultMaxSize   = 100
+	defaultMaxSize = 100
 )
 
+// BackupTimeformat time format for filename
+var BackupTimeFormat = "2006-01-02T15-04-05.000"
```

Comment:
Global variables are evil.  If you want to be able to modify this, it should be part of the Logger struct, so that it only affects the single instance.  Then you'd probably also want to expose it as DefaultBackupFormat for when the user doesn't specify a format.